.vscode/cspell.json

```diff
@@ -233,6 +236,7 @@
     "plex",
     "pluggy",
     "popen",
+    "prebuilts",
```

Comment:
Not sure if this fits better under the form recognizer specific overrides.

Reply:
I could see `prebuilts` being used in other AI libraries so I think it makes sense on the global list. The others in this list seem FR specific though and could probably go under the FR ignore list below.